src/display/editor/freetext.js

```diff
@@ -224,12 +224,9 @@ class FreeTextEditor extends AnnotationEditor {
     };
     const savedFontsize = this.#fontSize;
     this.addCommands({
-      cmd: () => {
-        setFontsize(fontSize);
-      },
-      undo: () => {
-        setFontsize(savedFontsize);
-      },
+      cmd: setFontsize.bind(this, fontSize),
```

Comment:
Here, and elsewhere in the patch, what's the purpose of using `bind` like this?
Given that e.g. `setFontsize` is an arrow function it already binds the "correct" scope at its creation and using `bind` is thus a no-op.